README.rst

```diff
@@ -1,22 +1,107 @@
-Examples of web apps for the following setups:
-
-1. aurelia (aurelia as MVC, webpack packaging and babel transpiling)
-2. backbone (grunt for building, no ecma2015 and higher)
-3. react (react.js, webpack packaging and babel transpiling)
-4. webpack (jquery only, webpack packaging and babel transpiling)
+This template repository contains examples of OMERO.web apps for the following setups:
 
+1. minimal example (no JavaScript dependencies or build steps)
+2. react (react.js, created using create-react-app)
 
+We will run the simplest example app below. This is called ``minimal_webapp``.
 
+See the ``react-webapp/README`` for details on building and installing that app.
 
 Requirements
 ============
 
-* OMERO 5.4.x or newer (incl. web).
-* Node.js 6 or higher (incl. npm).
+You can install and run the ``omero-web`` server in a python
+virtual environment or via Docker.
+
+The ``react-webapp`` example app uses Node.js
+to build the JavaScript app and requires Node.js 6 or higher.
+
+
+Clone this repository
+---------------------
+
+Clone this examples repo to a location of your choice:
+
+::
+
+    $ git clone git@github.com:will-moore/omero-web-apps-examples.git
+
+
+Run your app with locally-installed OMERO.web
+---------------------------------------------
+
+If you have installed OMERO.web locally in a virtual environment as
+`described here <https://docs.openmicroscopy.org/latest/omero/developers/Web/Deployment.html>`_,
+you can ``pip install`` the example apps. NB: all examples apart from the ``minimal_webapp`` also
+require a JavaScript install and build steps. See the README for each example for more details.
+
+To install ``minimal_webapp``::
+
+    # within your python venv:
+    $ cd minimal-webapp
+    $ pip install -e .
+
+You also need to add your app to the :property:`omero.web.apps` setting:
+
+NB: Here we use single quotes around double quotes.
+
+::
+
+    $ bin/omero config append omero.web.apps '"minimal_webapp"'
+
+Now restart your ``omero-web`` server and go to
+`http://localhost:4080/minimal_webapp/ <http://localhost:4080/minimal_webapp/>`_
+in your browser.
+
+
+Run your app with OMERO.web in a Docker container
+-------------------------------------------------
+
+The following walk-through uses `omero-web-docker <https://github.com/ome/omero-web-docker/>`_
+to run the app. Here we add ``minimal_webapp`` to the installed apps and map the
+app directory to the ``site-packages`` directory in the Docker instance so that
+python can import our ``minimal_webapp`` module.
+
+::
+
+    # You need to be in the project directory for this to work.
+    # cd omero-web-apps-examples/
+
+    # The OMERO server we want to connect to.
+    $ host=demo.openmicroscopy.org
+
+    # Path to the python module for the app.
+    $ appdir=$(pwd)/minimal-webapp/minimal_webapp
+
+    # Location within Docker instance we want to link the app, so it can be imported.
+    $ docker_appdir=/opt/omero/web/venv/lib/python2.7/site-packages/minimal_webapp
```

Comment:
Python 3

Reply:
This is working for me at the moment. I guess we are still using OMERO 5.5.1 with the last non-dev release of omero-web-docker and it will need updating.

Reply:
Yup. You'd need to hard code the `5.6` images as we've done a couple of times.